plugin/pkg/scheduler/api/validation/validation_test.go

```diff
@@ -50,3 +50,18 @@ func TestValidatePriorityWithNegativeWeight(t *testing.T) {
 		t.Errorf("Expected error about priority weight not being positive")
 	}
 }
+
+func TestValidateExtenderWithNonZeroWeight(t *testing.T) {
```

Comment:
s/NonZero/NonNegative

Sorry, I should mention this earlier, I prefer that we use `NonNegative` here as the `Extender` only  have `negative` and `non negative` case.